src/plone/restapi/profiles/default/rolemap.xml

```diff
@@ -3,5 +3,9 @@
     <permission name="plone.restapi: Use REST API" acquire="True">
       <role name="Anonymous"/>
     </permission>
+    <permission name="plone.restapi: Access Plone vocabularies" acquire="True">
```

Comment:
@sneridagh I had some doubts about the naming "access" and not "view". Though, looking at all the permissions we have in Plone there is no way to be consistent with all the inconsistency we already have...